### PR TITLE
Disable audio when tabbing out

### PR DESCRIPTION
**Pull request by medsouz**
Implementation of #1758

### CODE REVIEW

File: src/config.c [hunk 1]
```diff
@@ -206,7 +206,8 @@ config_property_definition _interfaceDefinitions[] = {
 config_property_definition _soundDefinitions[] = {
 	{ offsetof(sound_configuration, title_music),						"title_music",					CONFIG_VALUE_TYPE_UINT8,		2,								NULL					},
 	{ offsetof(sound_configuration, sound),								"sound",						CONFIG_VALUE_TYPE_BOOLEAN,		true,							NULL					},
-	{ offsetof(sound_configuration, ride_music),						"ride_music",					CONFIG_VALUE_TYPE_BOOLEAN,		true,							NULL					},
+	{ offsetof(sound_configuration, ride_music),						"ride_music",					CONFIG_VALUE_TYPE_BOOLEAN,		false,							NULL					},
```
Comment (IntelOrca) [inline src/config.c]: Why did you change the default value for ride_music to false?

Comment (medsouz) [inline src/config.c]: I couldn't remember if it was in the original game or not so I just left it disabled by default. Want me to change it to true?

**Comment by Gymnasiast** [comment]
You added strings to the wrong language file. You should add new strings to english_uk.txt and report the added strings on Localisation.

**Comment by medsouz** [comment]
Do you want me to take it out of english_us or just add it to english_uk?

**Comment by Gymnasiast** [comment]
US English is maintained at Localisation, so you should take it out of this PR, yes.

**Comment by medsouz** [comment]
Aaaaaand GitHub Atom ate the newlines. I should have checked the diff before commit, redoing it.

**Comment by medsouz** [comment]
Everything should be fixed now, sorry for the mistakes.
